src/bin/jp2/opj_compress.c

```diff
@@ -1535,6 +1538,31 @@ static void info_callback(const char *msg, void *client_data) {
     fprintf(stdout, "[INFO] %s", msg);
 }
 
+OPJ_FLOAT64 opj_clock(void) {
```

Comment:
Is there a reason why you did not use opj_clock.c and opj_clock.h already defined in the repo ? If possible, I would prefer these functions to be used rather than rewriting them in opj_(de)compress. Could you update the PR in that way ?